test/travis-ci.sh

```diff
@@ -200,7 +200,8 @@ elif [[ $1 == "GJS" ]]; then
     # Build and test the latest commit (merged or from a PR) of Javascript Bindings for GNOME
     echo
     echo '-- gjs build --'
-    cd ../gjs
+    cp -r ../gjs ~/jhbuild/checkout/gjs
+    cd ~/jhbuild/checkout/gjs
```

Comment:
Due `E: The current directory is not in the checkout root '/root/jhbuild/checkout/' `, this is needed.

It that Ok to you?